openedx/core/djangoapps/content/learning_sequences/api/processors/schedule.py

```diff
@@ -58,13 +60,20 @@ def inaccessible_sequences(self, full_course_outline):
         Sequences are inaccessible, regardless of the individual Sequence start
         dates.
         """
+        if self._days_early_for_beta is not None:
+            start_offset = timedelta(days=self._days_early_for_beta)
+        else:
+            start_offset = timedelta(days=0)
```

Comment:
I think this offset should only be applied for users in the beta group for the course.

Reply:
That is what I suspected. How to I retrieve that information? I will see what I can find on my own, but some pointers would be helpful.

Reply:
I would need to look that up myself, but from a quick look I think you should check out `CourseBetaTesterRole`. You can test if a user is a beta tester using `CourseBetaTesterRole(course_id).has_user(user)`. If they do then apply these modifications. 

Note that according to the [discussion on the ticket](https://openedx.atlassian.net/browse/TNL-7261), you should check out how the system behaves right now. i.e. Is the start date just offset for such users, or do they see the regular date, but are still able to access the course.